### PR TITLE
Clarify how to disable warnings in deps

### PR DESCRIPTION
**Pull request by nyurik**

### CODE REVIEW

File: src/building/new-target.md [hunk 1]
```diff
@@ -141,6 +141,13 @@ to the `deny-warnings` setting in `config.toml`, the build may suddenly start
 to fail. To work around the warnings, you may want to disable `deny-warnings`
 in the config, or modify the dependency to remove the warnings.
```
Comment (jieyouxu) [inline src/building/new-target.md:142]: Suggestion: we should clarify this sentence:

> To workaround warnings, you may want to:
> - Modify the dependency to remove the warnings
> - Or for local development purposes, suppress the warnings by setting `deny-warnings = false` in `config.toml`.

File: src/building/new-target.md [hunk 2]
```diff
@@ -141,6 +141,13 @@ to the `deny-warnings` setting in `config.toml`, the build may suddenly start
...
 
+```toml
+# /config.toml
```
Comment (jieyouxu) [inline src/building/new-target.md:145]: Nit:

```suggestion
# config.toml
```

**Review by jieyouxu — APPROVED** [review]
Thanks for the PR, I left some suggestions

**Review by Noratrieb — APPROVED** [review]
Thanks for the PR, I *applied* some suggestions

**Pull request merged**